xds/internal/clients/xdsclient/xdsclient.go

```diff
@@ -221,7 +216,7 @@ func (c *XDSClient) Close() {
 // A non-nil error is returned if an xdsChannel was not created.
 func (c *XDSClient) getChannelForADS(serverConfig *ServerConfig, callingAuthority *authority) (*xdsChannel, func(), error) {
 	if c.done.HasFired() {
-		return nil, nil, ErrClientClosed
+		return nil, nil, errClientClosed
```

Comment:
Given that it's used exactly once, let's just inline it.

The comment on it became stale anyway.